Add App to Teams and Tenant/Add App to Teams or Tenant  using Graph API.ps1

```diff
@@ -0,0 +1,96 @@
+param(
```

Comment:
Please add Synopsis block, i.e. from this template for inline help. https://github.com/TeamsAdminSamples/PowerShell/blob/main/z_Templates/New-PowerShell.ps1